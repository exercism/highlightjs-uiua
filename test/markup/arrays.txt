[] [1 2 3] 1_2_3 [1_2 3_4]
