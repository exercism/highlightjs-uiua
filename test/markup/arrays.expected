<span class="hljs-punctuation">[</span><span class="hljs-punctuation">]</span> <span class="hljs-punctuation">[</span><span class="hljs-number">1</span> <span class="hljs-number">2</span> <span class="hljs-number">3</span><span class="hljs-punctuation">]</span> <span class="hljs-number">1</span><span class="hljs-punctuation">_</span><span class="hljs-number">2</span><span class="hljs-punctuation">_</span><span class="hljs-number">3</span> <span class="hljs-punctuation">[</span><span class="hljs-number">1</span><span class="hljs-punctuation">_</span><span class="hljs-number">2</span> <span class="hljs-number">3</span><span class="hljs-punctuation">_</span><span class="hljs-number">4</span><span class="hljs-punctuation">]</span>
